### ISSUE TITLE
action_link ajax call

### ISSUE THREAD

**Issue by markmcdonald51** [activescaffold/active_scaffold#385]
Hi,

I am pretty sure you can do this with AS but my first stab at this didnt seem to do what I expected. 
I just want to add in a column call State (aasm) which would account for the current state's transitions. For this event I would like to use the AS to dynamically display a link that reflects the new states. Not sure if this is possible with the current AS config but I'll ask. I just want this to make a json put get request that would then update this method. Something simple like:

    conf.action_links.add 'change_state', :label => [**current_state.events**], :page => false, :method => :get, :position => true, :type => :member, :parameters => {:format => 'json'}

Obviously this isn't possible as listed above as the label would need to be dynamic. I just want to re-render the row in place... which would contain the new events for that state.  Perhaps it would have to be a pull down select because most much of the time there are many events.  Anyway, I need to figure something like this out so I might start just diggin' into the core.  Perhaps what I am missing is a helper method that renders this? 

Thanks for the great module! It is very useful and I plan to share anything I do add.

**Comment by scambra** [comment]
Please, doesn't use github issues for support requests, there is a google group for that.

For your question, check advanced topics in wiki, specifically this page: https://github.com/activescaffold/active_scaffold/wiki/Dynamic-action-links-groups

**Comment by markmcdonald51** [comment]
Hi Scambra.

Just want to say thanks for the help and that the link you provided in your comment on this request sure did help! I was able to get the AASM (https://github.com/aasm/aasm)totally working with th GUI which I have since turned into a library so it may be used on many of my project. I plan to turn it into a simple gem so other might use this wonderful gem (AASM). Anyway, just wanted to thank you yet again. This really is a great utlity for CMS and so many other things!

cheers,
mark
